apps/react/src/features/genres/components/GenreCard/GenreCard.tsx

```diff
@@ -11,11 +13,19 @@ interface Props {
 }
 
 /** Card with genre data. */
-const GenreCardComponent: FC<Props> = ({ genre }) => (
-	<div className={styles.card}>
-		<h2>{genre.name}</h2>
-		<span>Id - {genre.id}</span>
-	</div>
-);
+const GenreCardComponent = forwardRef<HTMLLIElement | null, Props>(({ genre }, forwardedRef) => (
+	<>
+		<ListItem ref={forwardedRef} alignItems="flex-start" className={styles.card}>
+			<ListItemText
+				primary={genre.name}
+				secondary={
+					<Typography sx={{ display: 'inline' }} component="span" variant="body2" color="text.primary">
+						{GenreType.toReadable(genre.type)}
+					</Typography>
+				}
+			/>
+		</ListItem>
+	</>
+));
```

Comment:
A fragment in this case would be redundant
```suggestion
const GenreCardComponent = forwardRef<HTMLLIElement | null, Props>(({ genre }, forwardedRef) => (
		<ListItem ref={forwardedRef} alignItems="flex-start" className={styles.card}>
			<ListItemText
				primary={genre.name}
				secondary={
					<Typography sx={{ display: 'inline' }} component="span" variant="body2" color="text.primary">
						{GenreType.toReadable(genre.type)}
					</Typography>
				}
			/>
		</ListItem>
));
```